src/Telnyx.net/Services/Wireless/SimCards/SimCardEnableService.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace Telnyx.net.Services.Wireless.SimCards
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Telnyx.net.Entities.Wireless.SimCards;
+
+    /// <summary>
+    /// SimCardsEnableService
+    /// </summary>
+    public class SimCardEnableService : Service<SimCardRecord>,
+        INestedCreatableWithIdInMid<SimCardRecord, BaseOptions>
```

Comment:
Don't have to use this interface but also can override the PostPath property of Service.cs for the post path url

Reply:
i have addressed this.